python/ray/runtime_env/runtime_env.py

```diff
@@ -545,6 +549,9 @@ def py_container_run_options(self) -> List:
             return None
         return self["container"].get("run_options", [])
 
+    def image_uri(self) -> Optional[str]:
```

Comment:
Line 367 makes `container` mutual exclusive to other rt envs. I think `image_uri` also wants that?

Reply:
makes sense!